data/companies.json

```diff
@@ -496,6 +496,22 @@
         "youtube": "https://www.youtube.com/channel/UCNG0LqHlyecBNCcIAnAMtXw"
       }
     },
+    {
+      "name": "TechLekh Services Pvt. Ltd.",
+      "establishedAt": 1420099200,
```

Comment:
Have you checked the authenticity of this?

Reply:
yes